tools/sdk/ld/eagle.app.v6.common.ld.h

```diff
@@ -229,6 +247,35 @@ SECTIONS
     /* Fundamental type info */
     *(.rodata._ZTIPKc .rodata._ZTIc .rodata._ZTIv .rodata._ZTSv .rodata._ZTSc .rodata._ZTSPKc .rodata._ZTSi .rodata._ZTIi)
 
+    /* std::make_shared */
+    *(.rodata._ZZNSt19_Sp_make_shared_tag5_S_tiEvE5__tag)
+
+#ifdef FREE_MORE_DRAM
+    *(.sdk.version)
+    *libc.a:lib_a-dtoa.o                (.rodata .rodata.*)
+    *libc.a:lib_a-gdtoa-gethex.o        (.rodata .rodata.*)
+    *libc.a:lib_a-impure.o              (.rodata .rodata.*)
+    *libc.a:lib_a-lcltime.o             (.rodata .rodata.*)
+    *libc.a:lib_a-lnumeric.o            (.rodata .rodata.*)
+    *libc.a:lib_a-locale.o              (.rodata .rodata.* .data)
+    *libc.a:lib_a-mprec.o               (.rodata .rodata.*)
+    *libc.a:lib_a-nano-svfprintf.o      (.rodata .rodata.*)
+    *libc.a:lib_a-nano-svfscanf.o       (.rodata .rodata.*)
+    *libc.a:lib_a-nano-vfprintf_float.o (.rodata .rodata.*)
+    *libc.a:lib_a-nano-vfprintf.o       (.rodata .rodata.*)
+    *libc.a:lib_a-nano-vfscanf_float.o  (.rodata .rodata.*)
+    *libc.a:lib_a-nano-vfscanf_i.o      (.rodata .rodata.*)
+    *libc.a:lib_a-rand.o                (.rodata .rodata.*)
+    *libc.a:lib_a-strtod.o              (.rodata .rodata.*)
+    *libc.a:lib_a-tzset_r.o             (.rodata .rodata.*)
+    *libc.a:lib_a-tzvars.o              (.rodata .rodata.*)
+    *liblwip2-536.a:                    (.rodata .rodata.memp_pools .rodata.tcp_pcb_lists .rodata.dns_mquery_v4group .rodata.ip_addr_broadcast .rodata.ip_addr_any)
+    *liblwip2-1460.a:                   (.rodata .rodata.memp_pools .rodata.tcp_pcb_lists .rodata.dns_mquery_v4group .rodata.ip_addr_broadcast .rodata.ip_addr_any)
```

Comment:
?
no -feat, no ipv6 variant. unless intentional.
not really sure every .rodata is readable, though. doesn't this also need exception trap for unaligned reads e.g. when we try to read u16 or u8 / strings?

Reply:
I added feat and v6

the FREE_MORE_DRAM is not stable for everyone.

Reply:
I removed FREE_MORE_DRAM because its need to trace from non 32bit load handler.